internal/operators/cnv/config.go

```diff
@@ -9,6 +9,15 @@ type SupportedGPUsDecoder map[string]bool
 type Config struct {
 	// List of supported GPUs: https://issues.redhat.com/browse/CNV-7749
 	SupportedGPUs SupportedGPUsDecoder `envconfig:"CNV_SUPPORTED_GPUS" default:"10de:1db6,10de:1eb8"`
+
+	// CNV operator namespace `openshift-cnv` for downastream and `kubevirt-hyperconverged` for upstream
+	Namespace string `envconfig:"CNV_OPERATOR_NAMESPACE" default:"openshift-cnv"`
+
+	// CNV operator source from where we fetch HCO image for downstream `redhat-operators` for upstream `community-operators`
+	Source string `envconfig:"CNV_OPERATOR_SOURCE" default:"redhat-operators"`
+
+	// CNV operator source name used to identify HCO for downstream `kubevirt-hyperconverged` for upstream `community-kubevirt-hyperconverged`
+	SourceName string `envconfig:"CNV_OPERATOR_SOURCE_NAME" default:"kubevirt-hyperconverged"`
```

Comment:
Seems that they have only 2 possible values.. I think that using an environment variable for that is unnecessary since I don't see anyone that would override those 2 defaults.
 Alternatively, we should just support `CNV_MODE`? Then we could have a struct with the values for d/s and for u/s accordingly.

Reply:
Sounds like good idea. Will change